src/main/java/org/swisspush/redisques/RedisQues.java

```diff
@@ -181,13 +187,11 @@ public void start() {
         httpRequestHandlerUserHeader = modConfig.getHttpRequestHandlerUserHeader();
         queueConfigurations = modConfig.getQueueConfigurations();
 
-        this.redisClient = RedisClient.create(vertx, new RedisOptions()
-                .setHost(redisHost)
-                .setPort(redisPort)
-                .setAuth(redisAuth)
-                .setEncoding(redisEncoding));
-
-        this.luaScriptManager = new LuaScriptManager(redisClient);
+        this.redisClient = new RedisClient(vertx, new RedisOptions().setConnectionString("redis://" + redisHost + ":" + redisPort).setPassword(redisAuth));
+        //TODO: vertx4
+        //       .setEncoding(redisEncoding));
```

Comment:
I look around there no way to set encoding, that why I put a TODO here.

Reply:
It was set to

this.redisEncoding = "UTF-8";

before, let's find out to what we get now.

Reply:
read here, new version doesn't serialize binary data any more. 
https://groups.google.com/g/vertx-dev/c/xBElx1yu3pg

Reply:
OK, I think it will be fine. Let's keep in mind that we should verify handling of binary data in gateleen once we are through (if not already integration tested).